doc/develop/sca/eclair.rst

```diff
@@ -0,0 +1,154 @@
+.. _eclair:
+
+ECLAIR support ( Experimental )
+###############################
+
+Bugseng `ECLAIR <https://www.bugseng.com/eclair/>`__ is a certified
+static analysis tool and platform for software verification.
+Applications range from coding rule validation, with a
+particular emphasis on the MISRA and BARR-C coding standards, to the
+computation of software metrics, to the checking of independence and
+freedom from interference among software components, to the automatic
+detection of important classes of software errors.
+
+.. important::
+
+   ECLAIR is a commercial tool, and it is not free software.
+   You need to have a valid license to use it.
+
+.. warning::
+
+   The Integration of the tool is experimental right now!
+   Following known features/use may lead to issues or differen analysis results:
+
+   * Using it without :ref:`west <west>` build would not work
+   * Passing extra configuration files or configurations to the build like ``-DEXTRA_CONF_FILE``
+     or ``-- -DCONFIG_BAR=y`` may end up in not complete results.
```

Comment:
Please also described that sysbuild will not work (at least that's my assumption, you might want to verify)